Improvement on #1585

linuxbckp opened the pull request.

opbld32 commented:
Docs Build status updates of commit _[5ad6a70](https://github.com/linuxbckp/WSL_develop20220816/commits/5ad6a70afcfdf5816531ff3e1ed42a7366bed1e3)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[WSL/about.md](https://github.com/linuxbckp/WSL_develop20220816/blob/main/WSL/about.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/windows/wsl/about?branch=pr-en-us-1586) | [Details](#user-content-e66687d45d8894c53593dcdbeeba3d3c-ee23a24f)
[WSL/media/WSLLogo.svg](https://github.com/linuxbckp/WSL_develop20220816/blob/main/WSL/media/WSLLogo.svg) | :white_check_mark:Succeeded | |

<a id="e66687d45d8894c53593dcdbeeba3d3c-ee23a24f"></a>
### [WSL/about.md](https://github.com/linuxbckp/WSL_develop20220816/blob/main/WSL/about.md)
 - **Line 11, Column 1**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../media/WSLLogo.svg'.`````
 - **Line 13, Column 1**: **[Warning: multiple-h1s - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/multiple-h1s?branch=main)]** `````Multiple H1s(H1 'What is the Windows Subsystem for Linux?') are not allowed. You can only have one top-level heading.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/679c7023-7559-8831-3c77-6434edf8d58b/202208161226041921-1586/BuildReport?accessString=801621ef0ce30e89c9a17dea3f881f6280631b4a2e3b6e70a8a3b15b87b03f60).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/679c7023-7559-8831-3c77-6434edf8d58b?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

linuxbckp commented:
The font of 'WSL' is named 'Asset'. We may find the license at https://www.fontsquirrel.com/fonts/asset.
------------------------------------------------
The recent version of the license is as below:
SIL Open Font License v1.10
This license can also be found at this permalink: https://www.fontsquirrel.com/license/asset

Copyright (c) 2011 by Sorkin Type Co ([www.sorkintype.com](http://www.sorkintype.com/)),
with Reserved Font Name “Asset”.

This Font Software is licensed under the SIL Open Font License, Version 1.1.
This license is copied below, and is also available with a FAQ at: http://scripts.sil.org/OFL

—————————————————————————————-
SIL OPEN FONT LICENSE Version 1.1 - 26 February 2007
—————————————————————————————-

PREAMBLE
The goals of the Open Font License (OFL) are to stimulate worldwide development of collaborative font projects, to support the font creation efforts of academic and linguistic communities, and to provide a free and open framework in which fonts may be shared and improved in partnership with others.

The OFL allows the licensed fonts to be used, studied, modified and redistributed freely as long as they are not sold by themselves. The fonts, including any derivative works, can be bundled, embedded, redistributed and/or sold with any software provided that any reserved names are not used by derivative works. The fonts and derivatives, however, cannot be released under any other type of license. The requirement for fonts to remain under this license does not apply to any document created using the fonts or their derivatives.

DEFINITIONS
“Font Software” refers to the set of files released by the Copyright Holder(s) under this license and clearly marked as such. This may include source files, build scripts and documentation.

“Reserved Font Name” refers to any names specified as such after the copyright statement(s).

“Original Version” refers to the collection of Font Software components as distributed by the Copyright Holder(s).

“Modified Version” refers to any derivative made by adding to, deleting, or substituting—in part or in whole—any of the components of the Original Version, by changing formats or by porting the Font Software to a new environment.

“Author” refers to any designer, engineer, programmer, technical writer or other person who contributed to the Font Software.

PERMISSION & CONDITIONS
Permission is hereby granted, free of charge, to any person obtaining a copy of the Font Software, to use, study, copy, merge, embed, modify, redistribute, and sell modified and unmodified copies of the Font Software, subject to the following conditions:

1) Neither the Font Software nor any of its individual components, in Original or Modified Versions, may be sold by itself.

2) Original or Modified Versions of the Font Software may be bundled, redistributed and/or sold with any software, provided that each copy contains the above copyright notice and this license. These can be included either as stand-alone text files, human-readable headers or in the appropriate machine-readable metadata fields within text or binary files as long as those fields can be easily viewed by the user.

3) No Modified Version of the Font Software may use the Reserved Font Name(s) unless explicit written permission is granted by the corresponding Copyright Holder. This restriction only applies to the primary font name as presented to the users.

4) The name(s) of the Copyright Holder(s) or the Author(s) of the Font Software shall not be used to promote, endorse or advertise any Modified Version, except to acknowledge the contribution(s) of the Copyright Holder(s) and the Author(s) or with their explicit written permission.

5) The Font Software, modified or unmodified, in part or in whole, must be distributed entirely under this license, and must not be distributed under any other license. The requirement for fonts to remain under this license does not apply to any document created using the Font Software.

TERMINATION
This license becomes null and void if any of the above conditions are not met.

DISCLAIMER
THE FONT SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT OF COPYRIGHT, PATENT, TRADEMARK, OR OTHER RIGHT. IN NO EVENT SHALL THE COPYRIGHT HOLDER BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, INCLUDING ANY GENERAL, SPECIAL, INDIRECT, INCIDENTAL, OR CONSEQUENTIAL DAMAGES, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF THE USE OR INABILITY TO USE THE FONT SOFTWARE OR FROM OTHER DEALINGS IN THE FONT SOFTWARE.

Notes from Fontsquirrel about the heritage of this font:
https://fonts.google.com/specimen/Asset

linuxbckp commented:
Discuss this PR in #1585
